### ISSUE TITLE
Accessibility option - add Pause capability to Carousel by keyboard/user interaction not by mouse

### ISSUE THREAD

**Issue by baloo32** [coreui/coreui-angular#214]
The Carousel is great but it's use fails a Web Content Accessibility Guidelines (WCAG) 2 Level AA Conformance accessibility review due to the inability to  programmatically pause the carousel outside of on mouse hover.

If you set the `interval` value to 0, the code doesn't ever recheck the value if it is subsequently changed to a positive value again, so while it is possible to stop the transitions, there is no way to restart them?

**Comment by xidedix** [comment]
@baloo32  Thanks for the heads-up!
- Fixed in `5.3.12`
Please let us know if this works for you. If not, reopen the issue.

**Comment by baloo32** [comment]
Unfortunately after setting interval to 0, resetting back doesn't restart the timer.

![Image](https://github.com/user-attachments/assets/0b59c234-0aa6-47a9-8b66-0bbcd5c439bb)
Screen above shows interval setting (2000 - 2 seconds).

Clicking Pause sets the interval value to 0. Carousel does pause (sometimes, not everytime it advances one more slide before pausing).

![Image](https://github.com/user-attachments/assets/0faca127-a6a4-43ce-bb73-99dc8e7787e8)

Clicking Pause again resets the interval value back to 2000 however it doesn't resume transitions.

![Image](https://github.com/user-attachments/assets/607b2da7-765d-4f75-b633-2e61fa07e14d)

![Image](https://github.com/user-attachments/assets/ea455345-0ca1-45d3-8415-f6671b72b01b)

Don't have permissions to reopen this issue...

Thx

**Comment by baloo32** [comment]
@xidedix sorry, can't reopen this issue - no permission.

**Comment by xidedix** [comment]
@baloo32, it should work with version `5.3.14`. Please let us know if the issue persists.

**Comment by baloo32** [comment]
Hi; yes just updated to 5.3.14 and the "unpause" now works as expected.

Many thanks!
